server/leafnode_test.go

```diff
@@ -784,6 +784,73 @@ func TestLeafNodeLoop(t *testing.T) {
 	checkLeafNodeConnected(t, sa)
 }
 
+func TestLeafNodeLoopFromDAG(t *testing.T) {
+	// This test requires that we set the port to known value because
```

Comment:
I don't think the comment here matches the rest of the test?

Reply:
Yes you are right, I removed it and pushed the squashed version.